### PR TITLE
Add ability to specify domains, lists, clients and group names as arrays

### PR DESCRIPTION
**Pull request by DL6ER**
# What does this implement/fix?

This PR accompanies the related FTL PR https://github.com/pi-hole/FTL/pull/1629. It depends on https://github.com/pi-hole/AdminLTE/pull/2700 which should be merged first to ease the reviewing process.

**Related issue or feature (if applicable):** N/A

**Pull request in [docs](https://github.com/pi-hole/docs) with documentation (if applicable):** N/A

---
**By submitting this pull request, I confirm the following:** 

1. I have read and understood the [contributors guide](https://docs.pi-hole.net/guides/github/contributing/), as well as this entire template. I understand which branch to base my commits and Pull Requests against. 
2. I have commented my proposed changes within the code.
3. I am willing to help maintain this change if there are issues with it later.
4. It is compatible with the [EUPL 1.2 license](https://opensource.org/licenses/EUPL-1.1)
5. I have squashed any insignificant commits. ([`git rebase`](http://gitready.com/advanced/2009/02/10/squashing-commits-with-rebase.html))

## Checklist:

- [x] The code change is tested and works locally.
- [x] I based my code and PRs against the repositories `developmental` branch.
- [x] I [signed off](https://docs.pi-hole.net/guides/github/how-to-signoff/) all commits. Pi-hole enforces the [DCO](https://docs.pi-hole.net/guides/github/dco/) for all contributions
- [x] I [signed](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) all my commits. Pi-hole requires signatures to verify authorship
- [x] I have read the above and my PR is ready for review.

### CODE REVIEW

**Comment by rdwebdesign** [comment]
Do we need to merge development-v6 here?

**Comment by DL6ER** [comment]
I just did it - it never hurts and we should pull in the theme changes.

**Comment by rdwebdesign** [comment]
Yeah... the theme was broken.
Now it's working.

Thanks.

File: scripts/pi-hole/js/utils.js [hunk 1]
```diff
@@ -537,6 +537,81 @@ function hexDecode(string) {
...
+      ":</strong>\n";
+
+    // Loop over data.processed.errors and print "item: error"
+    for (const item in data.processed.errors) {
+      if (Object.prototype.hasOwnProperty.call(data.processed.errors, item)) {
+        let error = data.processed.errors[item].error;
+        if (error === "UNIQUE constraint failed: domainlist.domain, domainlist.type") {
+          // Replace the error message with a more user-friendly one
+          error = "Already present";
+        }
```
Comment (rdwebdesign) [inline scripts/pi-hole/js/utils.js]: Lists, clients and groups pages will also use this function.

What do you think about this to allow user-friendly error messages?

```suggestion
        // Replace some error messages with a more user-friendly one
        if (error.indexOf("UNIQUE constraint failed:") > -1) {
          error = "Already present";
        }
```

**Review by rdwebdesign — CHANGES_REQUESTED** [review]

**Comment by DL6ER** [comment]
Review comments addressed

**Review by rdwebdesign — APPROVED** [review]

**Pull request merged**